bugtool/cmd/root.go

```diff
@@ -53,7 +53,8 @@ If you are going to register a issue on GitHub, please
 only provide files from the archive you have reviewed
 for sensitive information.
 `
-	defaultDumpPath = "/tmp"
+	defaultDumpPath          = "/tmp"
+	ciliumAgentContainerName = "cilium-agent"
```

Comment:
Can we add a way to specify the container name as a flag? The container name might be different in different environments.

Reply:
@aanm: good point, I added a commit to the PR.